get_http.go

```diff
@@ -220,6 +262,44 @@ func (g *HttpGetter) parseMeta(r io.Reader) (string, error) {
 	}
 }
 
+func GetByteRange(u *url.URL) ([]string, error) {
+	// format can be bytes=startByte- OR bytes=startByte-endByte
+	// example URL string: "http://my/file.iso?ranged_request_bytes=5555-66666"
+	// or  "http://my/file.iso?ranged_request_bytes=5555-"
+	// example bytes_range_retval: []int{5555, 66666} or []int{5555}
+	bytes_range_retval := make([]string, 0)
```

Comment:
I'd remove this. Unless I'm mistaken, it isn't assigned anywhere and a `nil` response functionally is equivalent and more idiomatic for no result but also errors. This would also avoid an allocation (though that's really an unnecessary optimization and not really the point).